DFReader.py

```diff
@@ -755,19 +755,26 @@ def init_arrays(self, progress_callback=None):
         HEAD2 = self.HEAD2
         lengths = [-1] * 256
 
+        if self.data_len < 3:
```

Comment:
This doesn't look right.  Why would we do this?

mavlogdump of an empty file should produce zero output, I think.  Of course, it currently falls over with an mmap error, but that's a separate issue.

Reply:
Hmmn, I was trying to fix another problem I see. But you are right I will take it out